### PR TITLE
Replace the invalid open source link with working one.

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: common/perf-test-core/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ helps developers create their performance tests for their APIs.
 ## Contributing
 
 If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft
```
Comment (weshaggard) [inline common/perf-test-core/README.md:35]: I think we should probably replace all these sections with a comment similar to the root README.md (For details on contributing to this repository, see the [contributing guide](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md)"

**Review by weidongxu-microsoft — APPROVED** [review]

File: common/perf-test-core/README.md (hunk 1 above)
Comment (sima-zhu) [reply common/perf-test-core/README.md:35]: Root readme includes more information than this line. Do you think it is better to refer the README contributing [section](https://github.com/Azure/azure-sdk-for-java/blob/master/README.md#contributing)?

Comment (sima-zhu) [reply common/perf-test-core/README.md:35]: Checked several readmes, it seems different sdk have different wording here. I would prefer to make minimum changes and ask team to make it standard.

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-amqp-experimental/README.md
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-queue/README.md#contributing

Comment (weshaggard) [reply common/perf-test-core/README.md:35]: Honestly for each of the service level readme's I think it is better to link to the repo level contributing guide it is just a matter of whether we should ask the owners of these files to update them or if we should just do a sweep. Given this is very generic I suspect we should just do a sweep to update them.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**